### DISCUSSION TITLE
Ganache rabbit hole....

### DISCUSSION THREAD

**Discussion by toddabraham** [smartcontractkit/full-blockchain-solidity-course-py#37] [Q&A] [answered]
I seem to have fallen down a bit of a rabbit hole now with deploying Ganache to my remote server.  I think most of the problem is with me not quite grasping the architecture of the setup I'm attempting to bring forth.

My impression is that Ganache will run as a server on my remote machine, and I'll access the interface through a browser.  Is that correct?

If that's the case, then my first step should be to install Ganache-CLI?

Signed,
Dazzed / Confused :-)

**Comment by toddabraham** [comment, marked as answer]
Fixed it.  Seems there was some issue with my npm installation which seemed to resolve itself with npm audit fix.  I hate it when I fix things without fully understanding how / why it resolved. :-)

**Reply by PatrickAlphaC** [reply]
TLDR:
Run: `npm audit fix`

Nice work!
